src/app/environement/page.tsx

```
@@ -0,0 +1,9 @@
+import React from 'react'
+
+type Props = {}
+
+export default function lesson({}: Props) {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)